client/containers/DashboardSettings/CommunitySettings.tsx

```diff
@@ -222,6 +228,65 @@ const CommunitySettings = () => {
 						setDescription(evt.target.value.substring(0, 280).replace(/\n/g, ' '));
 					}}
 				/>
+				<InputField
+					label={
+						<span>
+							Journal Citation
+							<Tooltip
+								content={
+									<span>
+										When filled out, this field will be used as the journal
+										title for
+										<br />
+										issue collections in citations, PDF exports, and Crossref
+										deposits.
+										<br />
+										By default, the Community Title is used.
+									</span>
+								}
+								// @ts-expect-error ts-migrate(2322) FIXME: Type '{ children: Element; content: Element; toolt... Remove this comment to see the full error message
+								tooltipClassName="bp3-dark"
+							>
+								<Icon icon="info-sign" />
+							</Tooltip>
+						</span>
+					}
+					type="text"
+					placeholder="This will show up in citations and BibTex as your journal name"
+					value={citeAs}
+					onChange={(evt) => {
+						setCiteAs(evt.target.value);
+					}}
+				/>
+				<InputField
+					label={
+						<span>
+							Publisher
+							<Tooltip
+								content={
+									<span>
+										When filled out, this field will be used as the publisher
+										for book and
+										<br />
+										conference proceedings collections in
+										<br />
+										citations, PDF exports, and Crossref deposits.
+									</span>
```

Comment:
Same thing here, sorry. Sloppy copy by me. And same question for @3mcd about the use of line breaks.

```When filled out, this field will be used as the Publisher for Book and Conference Proceedings Collections in citations, PDF exports, and Crossref deposits.```